features/users.feature

```diff
@@ -269,6 +269,334 @@ Feature: Users
+        Given a user with forename "Adam", lastname "Adams", email "email5@ons.gov.uk", id "id_2" and password "Passw0rd!" exists in the database
+        And a user with forename "William", lastname "Williams", email "email9@ons.gov.uk", id "id_1" and password "Passw0rd!" exists in the database
+        And a user with forename "Mary", lastname "Martin", email "email7@ons.gov.uk", id "id_3" and password "Passw0rd!" exists in the database
+        And a user with forename "Adam", lastname "Williams", email "email10@ons.gov.uk", id "id_4" and password "Passw0rd!" exists in the database
+        And a user with forename "William", lastname "Adams", email "email11@ons.gov.uk", id "id_5" and password "Passw0rd!" exists in the database
+
+        And I am an admin user
+        When I GET "/v1/users?sort=forename,lastname:desc"
+        Then I should receive the following JSON response with status "200":
+            """
+            {
+              "PaginationToken": "",
+              "count": 5,
+              "users": [
+                                              {
+                  "active": true,
+                  "email": "email10@ons.gov.uk",
+                  "forename": "Adam",
+                  "groups": [],
+                  "id": "id_4",
+                  "lastname": "Williams",
+                  "status": "CONFIRMED",
+                  "status_notes": ""
+                },
+                {
+                  "active": true,
+                  "email": "email5@ons.gov.uk",
+                  "forename": "Adam",
+                  "groups": [],
+                  "id": "id_2",
+                  "lastname": "Adams",
+                  "status": "CONFIRMED",
+                  "status_notes": ""
+                },
+                {
+                  "active": true,
+                  "email": "email7@ons.gov.uk",
+                  "forename": "Mary",
+                  "groups": [],
+                  "id": "id_3",
+                  "lastname": "Martin",
+                  "status": "CONFIRMED",
+                  "status_notes": ""
+                },
+                                {
+                  "active": true,
+                  "email": "email9@ons.gov.uk",
+                  "forename": "William",
+                  "groups": [],
+                  "id": "id_1",
+                  "lastname": "Williams",
+                  "status": "CONFIRMED",
+                  "status_notes": ""
+                },
+                                {
+                  "active": true,
+                  "email": "email11@ons.gov.uk",
+                  "forename": "William",
+                  "groups": [],
+                  "id": "id_5",
+                  "lastname": "Adams",
+                  "status": "CONFIRMED",
+                  "status_notes": ""
+                }
+                ]
+            }
+            """
```

Comment:
what happens if someone puts in:

`/users?sort=dogs:asc`

Reply:
Good test ... it shouldn't fail just will not sort the query